inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/flink/enums/Constants.java

```diff
@@ -69,4 +80,27 @@ public class Constants {
 
     public static final String SEPARATOR = ":";
 
+    /**
+     * Generate the Job name through {@link ProcessForm}: <br/> 
+     * When the ProcessForm is {@link GroupResourceProcessForm}, the format of the job name is 'InLong-Sort-{Group ID}-{Stream ID}', 
+     * otherwise take the  {@link Constants#DEFAULT_SORT_JOB_NAME}: 'InLong-Sort-Job'. 
+     */
+    public static Function<ProcessForm, String> SORT_JOB_NAME_GENERATOR = (ProcessForm processForm) -> {
```

Comment:
The inlong group is already a unique identifier, so there is no need to generate a job name based on the stream?